client/src/pages/CreateAccount/OrganizationAccount/components/Installations/index.jsx

```diff
@@ -0,0 +1,249 @@
+    const [loading, setLoading] = useState(false);
+
+    const daysOfWeekOptions = [
+        { label: "Domingo", value: "Sunday" },
+        { label: "Segunda-feira", value: "Monday" },
+        { label: "Terça-feira", value: "Tuesday" },
+        { label: "Quarta-feira", value: "Wednesday" },
+        { label: "Quinta-feira", value: "Thursday" },
+        { label: "Sexta-feira", value: "Friday" },
+        { label: "Sábado", value: "Saturday" },
+    ];
+
+    const handleChange = (value) => {
+        updateAnswers((prev) => ({
+            ...prev,
+            ...value,
+        }));
+    };
+
+    const handleStepInfo = async () => {
+        try {
+            setLoading(true);
+            await form.validateFields();
+            nextStep();
+            setLoading(false);
+        } catch (error) {
+            setLoading(false);
+        }
+    };
+
+    const handleFieldChange = () => {
+        const dogQty = form.getFieldValue("dogQty");
+        const catQty = form.getFieldValue("catQty");
+        const rabbitQty = form.getFieldValue("rabbitQty");
+        const birdQty = form.getFieldValue("birdQty");
+        const otherQty = form.getFieldValue("otherQty");
+
+        const anyFieldFilled =
+            dogQty > 0 ||
+            catQty > 0 ||
+            rabbitQty > 0 ||
+            birdQty > 0 ||
+            otherQty > 0;
+
+        setIsAnyFieldFilled(anyFieldFilled);
+        return anyFieldFilled;
+    };
+
+    return (
+        <>
+            <div className={styles.stepFormTitle}>
+                <div>
+                    <i className="ri-tent-line ri-2x"></i>
+                    <h2 className="headline4">Instalações</h2>
+                </div>
+                <p className="body1">
+                    Descreva as instalações da organização. Esta seção é
+                    importante para entendermos melhor o ambiente onde os
+                    animais são cuidados e abrigados.
+                </p>
+            </div>
+            <Form
+                layout="vertical"
+                form={form}
+                style={{ width: "100%" }}
+                initialValues={answers}
+                onValuesChange={(value) => {
+                    handleChange(value);
+                }}
+                onFinishFailed={handleFieldChange}
+                onFinish={() => {
+                    const result = handleFieldChange();
+                    if (result) {
+                        handleStepInfo();
+                    }
+                }}
+            >
+                <Form.Item
+                    label="Qual número máximo de animais que podem ser acomodados
+                    simultaneamente?"
+                    required
+                >
+                    <div className={styles.inputGrid}>
+                        <Form.Item label="Número de cachorros:" name="dogQty">
+                            <InputNumber
+                                style={{ width: "100%" }}
```

Comment:
se conseguir transferir essa estilização inline pro arquivo de estilos é melhor, se for particularidade da lib não ter o mesmo resultado que o className beleza. mas valida isso.